src/commands/scaling/addScaleRule/IAddScaleRuleContext.ts

```diff
@@ -3,27 +3,28 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import type { ScaleRule } from "@azure/arm-appcontainers";
+import { ScaleRule } from "@azure/arm-appcontainers";
 import type { ExecuteActivityContext } from "@microsoft/vscode-azext-utils";
 import type { ContainerAppModel } from "../../../tree/ContainerAppItem";
 import type { IContainerAppContext } from "../../IContainerAppContext";
 
 export interface IAddScaleRuleContext extends IContainerAppContext, ExecuteActivityContext {
     // Make containerApp _required_
     containerApp: ContainerAppModel;
-
-    scaleRules: ScaleRule[];
+    parentResourceName: string;
```

Comment:
No, in general the parent resource name could be the container app name or the revision name.

Reply:
Potentially I could add `parentResource: ContainerAppItem | Revision` and just access the name property later when I need it.  Felt like overkill to tack the whole thing on at the time, but it would make it a lot less confusing for situations like this where parent is a bit ambiguous 🤔

Reply:
Maybe just a comment about that then? I personally felt like it was pretty confusing.